tests/unit/amo/test_searchUtils.js

```diff
@@ -265,4 +267,28 @@ describe(__filename, () => {
       expect(fixFiltersFromLocation(filters)).toEqual({ page });
     });
   });
+
+  describe('generate_threshold_params', () => {
+    it('generates all the required threshold parameters for a given param', () => {
+      const generated = generate_threshold_params({ param: 'foo' });
+
+      expect(generated).toEqual({
+        foo__gt: 'foo__gt',
+        foo__lt: 'foo__lt',
+        foo__gte: 'foo__gte',
+        foo__lte: 'foo__lte',
+        foo: 'foo',
+      });
+    });
+  });
+
+  describe('paramsToFilter', () => {
+    it('generates all the required threshold parameters for ratings and users', () => {
+      expect(paramsToFilter).toEqual({
```

Comment:
Maybe only assert one or two params like:

```
expect(paramsToFilter).toContain('users__gt');
expect(paramsToFilter).toContain('ratings__lte');
```

Reply:
instead of the convoluted `.toEqual` or in addition to?

Reply:
instead, we don't need to check all possible values I believe.